apps/fhir/bluebutton/views/generic.py

```diff
@@ -103,7 +104,17 @@ def fetch_data(self, request, resource_type, *args, **kwargs):
         # TODO: waffle flag enforced, to be removed after v2 GA
         if self.version == 2 and (not waffle.flag_is_active(request, 'bfd_v2_flag')):
             err = exceptions.NotFound("bfd_v2_flag not active.")
-            log_v2_blocked(request.user, request.path, request.auth.application, err)
+            log_dict = {"type": "v2_blocked",
```

Comment:
```suggestion
            waffle_event_logger = logging.getLogger(request, logging.AUDIT_WAFFLE_EVENT_LOGGER)
            log_dict = {"type": "v2_blocked",
```

Reply:
Since we have access to the `request` before the log event, lets make sure we are capturing that.